main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1949,7 +1950,8 @@ public boolean onMenuItemClick(MenuItem menuItem) {
 
     @Override
     public void onSupportActionModeStarted(ActionMode mode) {
-        if (mActionMode == null && selectedTextView != null) {
+        if (mSelectionModeActive == true && selectedTextView != null) {
```

Comment:
Please don't compare to `true`. `if (mSelectionModeActive && selectedTextView != null)` is enough.